pkg/cmd/release/create/create.go

```diff
@@ -213,14 +218,29 @@ func createRun(opts *CreateOptions) error {
 		}
 	}
 
+	var tagDescription string
+	if opts.RepoOverride == "" {
+		var tagSHA string
+		tagSHA, tagDescription, _ = gitTagInfo(opts.TagName)
+		// existingTag is a short cut to knowing that a tag already exists on the remote
+		if tagSHA != "" && !existingTag {
+			remoteExists, err := remoteTagExists(httpClient, baseRepo, tagSHA)
+			if err != nil {
+				return err
+			}
+			if !remoteExists {
+				return fmt.Errorf("tag %s exists locally but has not been pushed to %s, please push it before continuing", opts.TagName, ghrepo.FullName(baseRepo))
```

Comment:
Open to other wording here.

Reply:
I think the current wording is fine, but I would like to reiterate my previous point from the thread: what if I run into this but want to ignore my local tag? Maybe I have had created a local tag for testing, but I do not actually want to publish that tag, or I have pulled an unrelated tag of the same name from a different git remote (maybe from a fork). I wanted to imagine that I will be able to use `--target main` to explicitly ignore the local tag and create a new tag from `main`. Would that be a worthwhile idea?

Reply:
That is a good point, and I like the idea that when `--target` is specified we just ignore the local tag if it exists.